packages/react/src/components/Combobox/Combobox.tsx

```diff
@@ -413,10 +612,22 @@ const Combobox = forwardRef<HTMLDivElement, ComboboxProps>(
     return (
       <div
         id={id}
-        className={classnames('Combobox', className)}
+        className={classnames(
+          'Combobox',
+          { 'Combobox--multiselect': multiselect },
+          className
+        )}
         ref={comboboxRef}
       >
-        {name && <input type="hidden" name={name} value={formValue} />}
+        {name &&
+          formValues.map((formValue) => (
+            <input
+              type="hidden"
+              key={formValue}
```

Comment:
While not likely, it's possible a form value could be duplicated. We should try to make sure this key is always unique regardless of the form value.